### PR TITLE
Update k8s

### PR DESCRIPTION
**Pull request by fahad-rauf**

### CODE REVIEW

**Comment by stakater-user** [comment]
@fahad-rauf Yikes! You better fix it before anyone else finds out! [Build 1](https://jenkins-delivery.stakater.com/job/Stakater/job/Whitelister/job/PR-12/1/) has Failed!

**Comment by stakater-user** [comment]
@fahad-rauf Image is available for testing. ``docker pull stakater/whitelister:SNAPSHOT-PR-12-2``

File: internal/pkg/ipProviders/kube/kube.go [hunk 1]
```diff
@@ -34,6 +34,9 @@ func (k *Kube) Init(params map[interface{}]interface{}) error {
 		return err
 	}
 
+	if k == nil {
+		return errors.New("Missing Kube From Port")
```
Comment (rasheedamir) [inline internal/pkg/ipProviders/kube/kube.go:38]: i think the error msg should be different in this case

**Comment by stakater-user** [comment]
@fahad-rauf Image is available for testing. ``docker pull stakater/whitelister:SNAPSHOT-PR-12-3``

**Comment by fahad-rauf** [comment]
@ahmedwaleedmalik should i update the version as well? in the .version file?

**Comment by ahmedwaleedmalik** [comment]
> @ahmedwaleedmalik should i update the version as well? in the .version file?
Yes please.

**Comment by stakater-user** [comment]
@fahad-rauf Image is available for testing. ``docker pull stakater/whitelister:SNAPSHOT-PR-12-4``

**Pull request merged**